### PR TITLE
CHAT-307 Use tests to document $.created augmentation

### PR DESCRIPTION
**Pull request by ianjennings**
Demonstrates how to get the created user and chat from the $.created event. The second parameter is not documented anywhere.

### CODE REVIEW

File: test/integration/main.test.js [hunk 1]
```diff
@@ -607,7 +621,6 @@ describe('remote chat list', () => {
 
         setTimeout(() => {
             let newChatToNotify = new ChatEngineClone.Chat(newChannel);
-            // we dont need this, just gets linter to pass
```
Comment (dogaan) [inline test/integration/main.test.js]: 😂

**Comment by dogaan** [comment]
Ah! You're adding it to the docs?

**Comment by ianjennings** [comment]
@dogaan this is in regards to CHAT-307 - #4. Since it was confusing I wanted a code example *somewhere* that demonstrates how to use those features. It would be better served in the docs for sure, but at least now there's something to link to.

**Comment by dogaan** [comment]
LGTM!

**Pull request merged**